pages/tv/index.vue

```diff
@@ -1,50 +1,38 @@
-<template>
-	<TVHero
-		:cover="globals.featured.season.show.cover"
-		:logo="globals.featured.season.show.logo"
-		:title="`Episode ${globals.featured.episode_number}: ${globals.featured.title}`"
-		:description="globals.featured.description"
-		:buttons="heroButtons"
-	/>
-	<BaseContainer class="main">
-		<section class="categories">
-			<TVCategory v-for="category in categories" :key="category.id" :title="category.title" :shows="category.shows" />
-		</section>
-	</BaseContainer>
-</template>
-
 <script setup>
-import { createDirectus, rest, readItems, readSingleton } from '@directus/sdk';
+const { $directusTv, $readSingleton, $readItems } = useNuxtApp();
 
 const {
-	public: { tvUrl, baseUrl },
+	public: { baseUrl, tvUrl },
 } = useRuntimeConfig();
 
-const directusUrl = process.env.DIRECTUS_TV_URL || tvUrl;
-const directus = createDirectus(directusUrl).with(rest());
-
-const globals = await directus.request(
-	readSingleton('globals', { fields: ['og', { featured: ['*', { season: [{ show: ['*'] }] }] }] }),
-);
+const { data: globals } = await useAsyncData('tv-globals', () => {
```

Comment:
Conflicts with the `tv-global` call in the live page, we need a different key, e.g.

```suggestion
const { data: globals } = await useAsyncData('tv-globals-deep', () => {
```